package-lock.json

```diff
@@ -11,6 +11,9 @@
         "configs/*",
         "scripts"
       ],
+      "dependencies": {
+        "electron": "^30.4.0"
+      },
```

Comment:
This shouldn't be here and our electron update script should make sure it's cleaned up. If you used it and it failed to clean up, can you open a build failure ticket to investigate?

Reply:
That sounds good, I'll open a ticket, because I only used `nvm use 20` for setting up my node environment and then run `node scripts/update-electron.js 30.4`.